### PR TITLE
Moved phpunit tests for php 7.1 running on sqlite to drone

### PR DESCRIPTION
**Pull request by SergioBertolinSG**

### CODE REVIEW

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]
one jenkins step -> 3 drone steps - this is not what we did agree on

File: .drone.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ pipeline:
       - ./tests/drone/composer-install.sh
     when:
       event: [push, pull_request]
+      branch: [master, stable10, stable9.1, stable9]
```
Comment (DeepDiver1975) [inline .drone.yml]: not necessary - there is a global branches config now in master - please rebase

File: .drone.yml [hunk 2]
```diff
@@ -91,6 +92,21 @@ pipeline:
...
+    pull: true
+    group: test
+    environment:
+      - PHP_VERSION=${PHP_VERSION}
+      - DB_TYPE=${DB_TYPE}
+    commands:
+      - ./tests/drone/test-phpunit.sh
+    when:
+      event: [push, pull_request]
+      branch: [master, stable10, stable9.1]
```
Comment (DeepDiver1975) [inline .drone.yml]: not necessary

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Comment by DeepDiver1975** [comment]
failing jenkins is because of this line - https://github.com/owncloud/core/pull/29365/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R24

**Comment by SergioBertolinSG** [comment]
> failing jenkins is because of this line - https://github.com/owncloud/core/pull/29365/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R24

Why does that line make it fail? The PHPUnit 7.1/sqlite stage was creating that file?

**Comment by DeepDiver1975** [comment]
> Why does that line make it fail? The PHPUnit 7.1/sqlite stage was creating that file?

its a type - the generated file is not named sqlite ....

**Comment by DeepDiver1975** [comment]
... and please squash your commits

**Comment by SergioBertolinSG** [comment]
Failing in jenkins: 

Test report file /var/lib/jenkins/workspace/owncloud-core_core_PR-29365-WR66RMKOWSY7ODQJVRC4FL3OUNMRUJ6P3FKCZ6TMQYQSUSTM5HCA@3/tests/autotest-results-oci.xml was length 0

But it doesn't seem related with my changes.

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by SergioBertolinSG** [comment]
@DeepDiver1975 all the drone PR should be backported to stable10?

**Pull request merged**